climada/hazard/trop_cyclone.py

```diff
@@ -485,25 +485,17 @@ def _apply_knutson_criterion(self, chg_int_freq, scaling_rcp_year):
                 sel_cat_chg = np.isin(tc_cc.category, cat_chg_list) & bas_sel
```

Comment:
Some lines above this one (L476-477), there is an obsolete comment about cumulative frequencies that needs to be removed as well.

Reply:
Yes, thank you!